### PR TITLE
Add IT for SQL and AD plugin in windows building workflow

### PR DESCRIPTION
**Pull request by weicongs-amazon**
### Changes ###
Add IT for SQL and AD plugin in windows building workflow
### Testing ###
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/170567600 for AD
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/170605015 for SQL

### CODE REVIEW

File: .github/workflows/test-build-windows.yml [hunk 1]
```diff
@@ -209,3 +209,157 @@ jobs:
...
+          dir
+          .\gradlew.bat :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -D tests.rest.cluster=localhost:9200 -D tests.cluster=localhost:9200 -D tests.clustername=es-integrationtest -D https=true -D user=admin -D password=admin --stacktrace
+
+  Test-SQL-plugin:
+    needs: [build-es-artifacts, build-kibana-artifacts]
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Retrieve plugin tags
+        run:  echo ::set-env name=p_tag_sql::$(.github/scripts/plugin_tag.sh  opendistro-for-elasticsearch/sql)
```
Comment (peterzhuamazon) [inline .github/workflows/test-build-windows.yml:266]: A space

File: .github/workflows/test-build-windows.yml [hunk 2]
```diff
@@ -209,3 +209,157 @@ jobs:
...
+          .\gradlew.bat integTest -D tests.rest.cluster=localhost:9200 -D tests.cluster=localhost:9200 -D tests.clustername=es-integrationtest --stacktrace
+        
+        
+  Test-SQL-plugin-With-Security:
+    needs: [build-es-artifacts, build-kibana-artifacts]
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Retrieve plugin tags
+        run:  echo ::set-env name=p_tag_sql::$(.github/scripts/plugin_tag.sh  opendistro-for-elasticsearch/sql)
```
Comment (peterzhuamazon) [inline .github/workflows/test-build-windows.yml:321]: Another Space

File: .github/workflows/test-build-windows.yml (hunk 1 above)
Comment (weicongs-amazon) [reply .github/workflows/test-build-windows.yml]: Fixed

File: .github/workflows/test-build-windows.yml (hunk 2 above)
Comment (weicongs-amazon) [reply .github/workflows/test-build-windows.yml]: Fixed

File: .github/workflows/test-build-windows.yml [hunk 3]
```diff
@@ -209,3 +209,157 @@ jobs:
...
+          $OD_VERSION=$(python ./bin/version-info --od)
+          $S3_PACKAGE="odfe-"+$OD_VERSION+".zip"
+          dir
+          echo downloading zip from S3 
+          aws s3 cp s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/odfe-windows/staging/odfe-window-zip/$S3_PACKAGE .\
+          echo "unzipping $S3_PACKAGE"
+          unzip .\$S3_PACKAGE
+          echo "running es"
+          nohup .\$PACKAGE-$OD_VERSION\bin\elasticsearch.bat &
+          echo "Waiting for 2min"
```
Comment (gaiksaya) [inline .github/workflows/test-build-windows.yml:252]: Are we doing anything here to wait for 2min? I mean like in "sleep 2m"?

Comment (weicongs-amazon) [reply .github/workflows/test-build-windows.yml:252]: just copy from previous IT case. It's using "ping -n 120 127.0.0.1 >.\out.txt" for 2m sleeping.

Comment (peterzhuamazon) [reply .github/workflows/test-build-windows.yml:252]: Good catch. That was there in the previous tests as well.
It is probably the reason that led to failure cases in workflows.
Agree we can add some 120 seconds sleep on each test cases.

**Pull request merged**